packages/enzyme/src/ReactWrapper.jsx

```diff
@@ -1060,4 +1071,25 @@ if (ITERATOR_SYMBOL) {
   });
 }
 
+function privateWarning(prop, extraMessage) {
```

Comment:
this function seems like it could live in a separate file and be shared, and take the proto as an argument?